README.md

```diff
@@ -16,6 +16,10 @@ Set up [Vundle](https://github.com/VundleVim/Vundle.vim) then add `Plugin 'cespa
 
 Set up [vim-plug](https://github.com/junegunn/vim-plug). In your .vimrc, between the lines for `call plug#begin()` and `call plug#end()`, add the line `Plug 'cespare/vim-toml'`. Then run `:PlugInstall` from a fresh vim instance.
 
+### Janus
+
+Set up [janus](https://github.com/carlhuda/janus) then clone/submodule this repo into `~/.janus` and restart vim.
```

Comment:
s/janus/Janus/

s/then/and then/ (or else add a comma; either way, get rid of the comma splice).